src/components/Posts/ExpandedPostItem.tsx

```diff
@@ -32,23 +32,16 @@ import { Post } from "../../state/slices/postsSlice";
 
 const styles = {
   actionButton: {
```

Comment:
we should remove the hover background from the action buttons
```suggestion
 actionButton: {
     "&:hover": {
         backgroundColor: "transparent",
      }
```

Reply:
Yeah, good one!